### ISSUE TITLE
Unable to download SDL anymore in 2.5.9

### ISSUE THREAD

**Issue by eg-ndobrijevic** [apollographql/apollo-kotlin#4040]
**Question.**
Somewhere between 2.4.0 and 2.5.9 the ability to download SDL was removed. I looked over release notes and I can't see if and why it was removed?

**Comment by martinbonnin** [comment]
Hi 👋  Thanks for reaching out. Can you elaborate on how you download your SDL? What command are you running and what error you're seeing? Thanks!

**Comment by eg-ndobrijevic** [comment]
Reproducible using one of the sample apps projects. Added more details there: https://github.com/martinbonnin/apollo-android-samples/pull/2

**Comment by martinbonnin** [comment]
Thanks for the reproducer. Indeed this is a regression, sorry for that. We'll fix it for the next 2.x release but given that 2.x is in  maintenance mode, I'd recommend to move to 3.x . Or you can use the following workaround:

```kotlin
tasks.withType(ApolloDownloadSchemaTask::class.java).configureEach {
  doLast {
    val schema = file("src/main/schema.sdl")
    IntrospectionSchema(schema).toSDL(schema)
  }
} 
```

**Comment by martinbonnin** [comment]
See https://github.com/apollographql/apollo-kotlin/pull/4046 for a fix in the 2.x branch. 

This is already fixed in 3.x so if you have the opportunity, I'd consider switching to 3.x. If not, this will be in the next 2.x release.

**Comment by eg-ndobrijevic** [comment]
Thanks 👍  I couldn't find a release plan in [readme](https://github.com/apollographql/apollo-kotlin/blob/main/README.md#releases); what is your release schedule for 2.x?

**Comment by martinbonnin** [comment]
2.x is in maintenance mode. Bugs and security fixes will be released as needed. TBH there's not many changes in the `release-2.x` branch so it could take several weeks until next release.

**Comment by BoD** [comment]
The fix is in v2.5.12 which was just released.
